### DISCUSSION TITLE
What do you think about side emitting LEDs?

### DISCUSSION THREAD

**Discussion by andyfeller** [martinwoodward/octolamp#10] [Q&A]
In reviewing the specs, I thought about trying side emitting LEDs like [Adafruit NeoPixel LED Side Light Strip](https://www.adafruit.com/product/3635?gclid=CjwKCAiArY2fBhB9EiwAWqHK6jwy43RIplGl_CkpTi9HNFuwd6_ctYQ2t569PezCziElHmOYbX-fPxoCA4IQAvD_BwE) since they're designed to work with Arduino boards.  I was still confirming whether the connections would work, but love your thoughts, @martinwoodward 🫶

![image](https://user-images.githubusercontent.com/2089743/217986769-21d937f8-ca70-4c2d-83b2-f35f7eb4f004.jpeg)

**Comment by martinwoodward** [comment]
Connections will definitely work, however they are pretty thick so might be a tight fit and honestly you don't need them because of the diffusion of the white cover combined with the reflection of the white inner component.

The thin, non-waterproof ones are the way to go for this particular model I think and should be cheaper.  I went for [60 LED's per meter](https://amzn.to/3XwWnCT) ones so I could power it from a low power USB supply and not need to do anything fancy.  I was _really_ tempted by the 144 LED's per meter strips but you'll need two of those for this design and that's a lot more LED's to power so the electronics are a bit more complex.  Also even at 60 LED's per meter you have to turn the brightness down to get the colors of the light to show up on camera.
